Open Judge System/MongoDBPoC/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace MongoDBPoC
+{
+    using System;
+    using System.Collections.Generic;
+
+    using MongoDB.Bson;
+    using MongoDB.Driver;
+
+    public class Program
+    {
+        static void Main()
+        {
+            const string ConnectionString = "mongodb://localhost:27017";
+            var client = new MongoClient(ConnectionString);
+            IMongoDatabase db = client.GetDatabase("OJS");
+            var collection = db.GetCollection<SubmissionForProcessing>("submissionsForProcessing");
+
+            InsertSubmissions(collection);
```

Comment:
this.

Reply:
`static` methods can't be invoked with `this`